website/content/docs/upgrading/index.mdx

```diff
@@ -131,3 +131,9 @@ verifying functionality after each cluster upgrade.
 3. Finally, upgrade Cluster A. All clusters downstream of A will already be
    upgraded. It should be upgraded last, as it is a Performance Primary and a DR
    Primary.
+
+
+## Activity Log
+
+The field `default_report_months` can no longer be configured. Attempts to write
+a value to the field will not result in any change.
```

Comment:
I think we probably want to use a version-based doc, like this: https://github.com/hashicorp/vault/blob/main/website/content/docs/upgrading/upgrade-to-1.17.x.mdx

I don't think there's a pre-existing 1.18 upgrade guide, but if you take a look at the docs-related changes in https://github.com/hashicorp/vault/commit/0637f5e316709c7f2eb0c5e3d6ad7030a8ef9e55#diff-98be8c35d45b7eff158ef4d272e6c3092c4501a1a88db1ee907365316fd6bbd4 it should show you what's needed:

1. Add the new file/update version numbers.
2. Update the docs nav .json file

Reply:
Just added this!

Reply:
We probably only want it one place, so we might want to remove the change to this file.